composer.json

```diff
@@ -5,6 +5,7 @@
   "keywords": [ "slots" ],
   "license": "GPL-2.0-or-later",
   "require": {
+    "php": "^8.0",
```

Comment:
What do you think about using `>= 8.0` instead?

Reply:
Agree that >= 8.0 should probably be used given that that is what is done in extension.json and we should be consistent one way or the other